demochatChart/app.rc.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: extensions/v1beta1
```

Comment:
@amir-codefresh can you explain why this deployment needed? 
And its not used helm concepts at all, all values are hard-coded